cypress/support/step_definition/tldraw/tldrawSteps.spec.js

```diff
@@ -0,0 +1,28 @@
+import { Then, When } from "@badeball/cypress-cucumber-preprocessor";
+import { Tldraw } from "../../pages/tldraw/pageTldraw";
+
+const tldraw = new Tldraw();
+
+When("I click on the pencil tool", () => {
+	tldraw.selectPencilTool();
+});
+
+When("I click on the text tool", () => {
+	tldraw.selectTextTool();
+});
+
+When("I draw a line on the canvas", () => {
+	tldraw.drawLine(100,100,300,300);
```

Comment:
IMO its not an issue but I think it would be better if we gets all these test data i.e. x-y coordinates, test string from feature file or if not then atleast in the step it's described to get an idea about it. Currently it says that user draw a line but where or text was typed but what text user is typing.